Form/Handler/AuthorizeFormHandler.php

```diff
@@ -23,10 +24,15 @@ class AuthorizeFormHandler
     protected $request;
     protected $form;
 
-    public function __construct(FormInterface $form, Request $request)
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    public function __construct(FormInterface $form, RequestStack $requestStack)
```

Comment:
Sorry, this is not about PHP 5.4. It is about Symfony 2.3 (which is still maintained). I messed my comment